Sources/UIKitExtensions/UIDatePicker+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIDatePickerFucntions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import UIKit
+public extension ConfigurationSet where Base: UIDatePicker {
+    
+    func setDate(_ date: Date, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UIDatePicker: Base ) in
+            UIDatePicker.setDate(date, animated: animated)
```

Comment:
This function is only useful for existing UIDatePickers whose date change should be animated.
The date Property ist writable directly so there is no need for this function.